src/coreclr/nativeaot/Runtime/unix/UnwindHelpers.cpp

```diff
@@ -34,6 +34,9 @@ using libunwind::Registers_arm;
 #elif defined(TARGET_ARM64)
 using libunwind::Registers_arm64;
 using libunwind::CompactUnwinder_arm64;
+#elif defined(TARGET_LOONGARCH64)
+using libunwind::Registers_loongarch;
+//using libunwind::CompactUnwinder_loongarch64;  //TODO-LOONGARCH64
```

Comment:
AFAIK LA64 doesn't use compact unwinding, so this doesn't need to exist. It's only relevant for Apple platforms.